main.go

```diff
@@ -341,30 +353,57 @@ func main() {
 		log.Donef("Using system installed CocoaPods version")
 	}
 
-	// Check ruby version
-	// Run this logic only in CI environment when the ruby was installed via rbenv for the virtual machine
-	if os.Getenv("CI") == "true" && rubycommand.RubyInstallType() == rubycommand.RbenvRuby {
-		fmt.Println()
-		log.Infof("Check selected Ruby is installed")
-
+	if rubycommand.RubyInstallType() == rubycommand.RbenvRuby {
+		rubySelectStart := time.Now()
 		rubyInstalled, rversion, err := rubycommand.IsSpecifiedRbenvRubyInstalled(configs.SourceRootPath)
 		if err != nil {
-			log.Errorf("Failed to check if selected ruby is installed, error: %s", err)
+			log.Errorf("Failed to check if selected ruby is installed: %s", err)
 		}
 
-		if !rubyInstalled {
-			log.Errorf("Ruby %s is not installed", rversion)
+		// Check ruby version
+		// Run this logic only in CI environment when the ruby was installed via rbenv for the virtual machine
+		if os.Getenv("CI") == "true" {
 			fmt.Println()
+			log.Infof("Check selected Ruby is installed")
+
+			if !rubyInstalled {
+				log.Errorf("Ruby %s is not installed", rversion)
+				fmt.Println()
 
-			cmd := command.New("rbenv", "install", rversion).SetStdout(os.Stdout).SetStderr(os.Stderr)
-			log.Donef("$ %s", cmd.PrintableCommandArgs())
-			if err := cmd.Run(); err != nil {
-				log.Errorf("Failed to install Ruby version %s, error: %s", rversion, err)
+				cmd := command.New("rbenv", "install", rversion).SetStdout(os.Stdout).SetStderr(os.Stderr)
+				log.Donef("$ %s", cmd.PrintableCommandArgs())
+				if err := cmd.Run(); err != nil {
+					log.Errorf("Failed to install Ruby version %s, error: %s", rversion, err)
+				}
+			} else {
+				log.Donef("Ruby %s is installed", rversion)
 			}
-		} else {
-			log.Donef("Ruby %s is installed", rversion)
 		}
 
+		rubySelectDuration := time.Since(rubySelectStart)
+		isRequiredRubyInstalled, _, err := rubycommand.IsSpecifiedRbenvRubyInstalled(podfileDir)
+		if err != nil {
+			log.Errorf("Failed to check if selected ruby is installed: %s", err)
+		}
+
+		effectiveRubyVersion, err := command.New("rbenv", "global").RunAndReturnTrimmedOutput()
+		if err != nil {
+			log.Errorf("Failed to check global rbenv version: %w", err)
+		}
+		if isRequiredRubyInstalled {
+			effectiveRubyVersion = rversion
+		}
+
+		log.Infof("Sending analytics...")
```

Comment:
I don't think it's valuable in itself if we don't print anything else about what is sent. I usually don't print anything about the tracker in other steps.

Reply:
Yes, it was intended as a debug line only.